less/datepicker.less

```diff
@@ -60,18 +62,30 @@
 				}
 
 				&.current-day {
-					b, button {
+					span {
 						border: 1px solid #333;
+						b, button {
```

Comment:
Why has this border styling been transferred to the `span` vs the `b, button`? It appears to look ok like that, just was curious about the reasoning. Might as well remove this `b, button` definition then since it is empty.